cdk/lib/telemetry-stack.ts

```diff
@@ -126,7 +126,7 @@ export class TelemetryStack extends GuStack {
 		const commonLambdaParams: Omit<FunctionProps, 'code'> = {
 			runtime: Runtime.NODEJS_20_X,
 			memorySize: 128,
-			timeout: Duration.seconds(5),
+			timeout: Duration.seconds(10),
```

Comment:
I don’t think there’s any explanation in the PR of the reason for increasing the timeout? (Happy with it, just would appreciate a written justification for our records.)

Reply:
Good point, I have added a section to the description for this.